compile-test/crmIntegrationTests/src_test/ch/ivyteam/integrationtest/WebTestOrderFormIT.java

```diff
@@ -34,7 +34,7 @@
 import com.codeborne.selenide.SelenideElement;
 import com.codeborne.selenide.WebElementCondition;
 
-@IvyWebTest(headless=false)
+@IvyWebTest(headless=true)
```

Comment:
the reason for flaky builds... if there is enough memory the web-driver with XVFB and the like works

Reply:
fyi @ivy-cst , seems like our investigation regarding the web-test capabilities in the end just comes back to the point that our build-nodes in most situations can run web-tests without headless mode... but under heavy load or lack of memory :shrug: they fail.